data/mapping/mapping.json

```diff
@@ -72,13 +65,12 @@
           }
         },
         "doctors": {
-          "type": "text",
-          "analyzer": "gp_name_analyzer",
-          "fields": {
-            "keyword": {
-              "type": "keyword",
-              "ignore_above": 256
-            }
+          "type": "nested",
```

Comment:
I understand the need for setting `doctors` to be `nested` but I'm not sure the reason for the transformation in the first instance.